### PR TITLE
Teems data update to include BigIP reg key

### PR DESCRIPTION
**Pull request by agupta49**
**Description**:  Teems data update to include BigIP reg key

**Changes Proposed in PR**:

**Fixes**: resolves #_Github issue id_

## General Checklist
- [x] Updated Added functionality/ bug fix in release notes
- [x] Added examples for new feature
- [x] Updated the documentation
- [x] Smoke testing completed

## CRD Checklist
- [x] Updated required CR schema

### CODE REVIEW

File: pkg/crmanager/postManager.go [hunk 1]
```diff
@@ -309,6 +309,38 @@ func (postMgr *PostManager) GetBigipAS3Version() error {
...
+// GetBigipRegKey ...
+func (postMgr *PostManager) GetBigipRegKey() (string, error) {
+	url := postMgr.getBigipRegKeyURL()
+	req, err := http.NewRequest("GET", url, nil)
+	if err != nil {
+		log.Errorf("Creating new HTTP request error: %v ", err)
+		return "", err
+	}
+
+	log.Infof("Posting GET BIGIP Reg Key request on %v", url)
```
Comment (sravyap135) [inline pkg/crmanager/postManager.go:321]: can we remove this log or have it in debug level ?

**Review by sravyap135 — APPROVED** [review]
LGTM

File: pkg/crmanager/postManager.go (hunk 1 above)
Comment (agupta49) [reply pkg/crmanager/postManager.go:321]: Yes, We can move it to debug level.

**Pull request merged**